javascript/javascript1/week4/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <title>PP</title>
+</head>
+<body>
+ 
+
+ <h1 style="font-size: 140px;"></h1>
+ <script src="Voice-assistant.js"></script>
```

Comment:
Usually, file names would be all lowercase and may include underscores `_` or dashes `-`, but no additional punctuation.